BUILD.md

```diff
@@ -24,3 +24,17 @@ tito tag
 ```
 tito build --rpm
 ```
+
+## Build openshift-ansible container image
+
+To build a container image of `openshift-ansible`:
+
+* Using standalone **Docker**:
+
+        cd openshift-ansible
+        docker build -t openshift/openshift-ansible .
+
+* Using an **OpenShift** build:
+
+        oc new-build docker.io/aweiteka/playbook2image~https://github.com/openshift/openshift-ansible
+        oc describe imagestream openshift-ansible
```

Comment:
Maybe "Once the build is finished, you can get more information about the built image with: ..."?

As for _feedback_, I think the best way is to use `oc logs`? Or IIRC there's a flag to new-build to follow the logs...